### PR TITLE
Funcsharp without covariance + bumped packages

### PR DESCRIPTION
**Pull request by KaliCZ**
## Description

Bumps Funscharp version.

Fixes #issue

## Type of change
- [ ] Bug fix.
- [ ] Feature.
- [ ] Consolidation.

## Checklist
- [ ] Is breaking change.
- [ ] Documentation updated.
- [ ] Tests included (please specify cases).

### CODE REVIEW

**Comment by jbrestan** [comment]
Hmm, this will leak the preview build of FuncSharp to all Mews.Fiscalization consumers. Do you want that? OTOH I don't know how else to do this than making this preview as well

File: src/Mews.Fiscalizations.All.props [hunk 1]
```diff
@@ -3,6 +3,6 @@
     <AddSyntheticProjectReferencesForSolutionDependencies>false</AddSyntheticProjectReferencesForSolutionDependencies>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="FuncSharp" Version="[6.0.0, 7)" />
+    <PackageReference Include="FuncSharp" Version="7.0.1" />
```
Comment (jbrestan) [inline src/Mews.Fiscalizations.All.props:6]: Shouldn't it be this?
```suggestion
    <PackageReference Include="FuncSharp" Version="[7.0.0-preview.1]" />
```

Comment (KaliCZ) [reply src/Mews.Fiscalizations.All.props]: It should. I was reverting the packages, so I forgot about this. 
I did use the correct one in Globalization though 😄

Comment (jbrestan) [reply src/Mews.Fiscalizations.All.props]: That's where I copied it from 😅

**Review by jbrestan — APPROVED** [review]

**Pull request merged**